examples/knowledge/cmd/root.go

```diff
@@ -28,13 +28,15 @@ import (
 
 	"github.com/indykite/indykite-sdk-go/grpc"
 	apicfg "github.com/indykite/indykite-sdk-go/grpc/config"
+	"github.com/indykite/indykite-sdk-go/ingest"
 	"github.com/indykite/indykite-sdk-go/knowledge"
 )
 
 var (
-	cfgFile string
-	client  *knowledge.Client
-	jsonp   = protojson.MarshalOptions{
+	cfgFile      string
+	client       *knowledge.Client
+	clientIngest *ingest.Client
```

Comment:
Same as above. Why in examples for Knowledge we have Ingest client?

Reply:
for the moment, I cannot create a delete all with only knowledge or only ingest